chart/ingress-operator/templates/ingress-operator-dep.yaml

```diff
@@ -1,5 +1,4 @@
-{{- $functionNs := default .Release.Namespace .Values.functionNamespace }}
-{{- if .Values.ingressOperator.create }}
+{{- if .Values.create }}
```

Comment:
Don't we always want to create these things now it's in it's own chart? (few files with this in)